scripts/__input_config_gamepads/__input_config_gamepads.gml

```diff
@@ -14,6 +14,10 @@
 //Input treats these two modes as mutually exclusive (come talk to us if you need to be able to swap at runtime)
 #macro INPUT_SWITCH_HORIZONTAL_HOLDTYPE  true
 
+//Whether to use the right hand sensor for motion data when using detached dual Joy-Cons as a pair
+//When <false>, the left hand Joy-Con sensor will be used for motion data instead 
+#macro INPUT_SWITCH_JOYCON_MOTION_RIGHT_HAND true
+
```

Comment:
Is left joycon the expected default?

Reply:
Examples I found default to pointing with right hand ala Wii Remote. Not sure if this looks wrong or you prefer the macro be `LEFT` and default false ?